frontend/src/util/index.ts

```diff
@@ -153,6 +153,12 @@ export const isConfigChange = (
   oldConfig: SearchConfig,
   newConfig: SearchConfig
 ): boolean => {
-  console.log(oldConfig.region, newConfig.region);
-  return JSON.stringify(oldConfig) !== JSON.stringify(newConfig);
+  const _old = { ...oldConfig };
+  const _new = { ...newConfig };
+  _old.utilization = 0;
+  _old.leaseLength = 0;
+  _new.utilization = 0;
+  _new.leaseLength = 0;
+
+  return JSON.stringify(_old) !== JSON.stringify(_new);
```

Comment:
[lodash's isEqual](https://lodash.com/docs/4.17.15#isEqual) might be helpful.
It's okay for a frontend engineer to understand the deep comparison by JSON.stringify trick. But it might be hard for other engineers.